src/main/scala/gestalt/Denotations.scala

```diff
@@ -0,0 +1,20 @@
+package scala.gestalt
+
+trait Denotations { self: Toolbox =>
+  type Denotation
+
+  implicit class DenotationOps(denot: Denotation) {
+    def name: String = Denotation.name(denot)
+    def info: Type = Denotation.info(denot)
+    def symbol: Symbol = Denotation.symbol(denot)
+  }
+
+  val Denotation: DenotationImpl
+  trait DenotationImpl {
+    def name(denot: Denotation): String
+
+    def info(denot: Denotation): Type
+
+    def symbol(denot: Denotation): Symbol
```

Comment:
This `symbol` and `Symbols` will be removed once we've enough experiments to show that `Denotation` suffices without symbols.